code/processes/dqe.q

```diff
@@ -64,13 +68,16 @@ chkcompare:{[runtype;idnum;params]
   a:d[`results] where not d[`procs]=params`compproc;                                                            /- obtain all the check returns
   procsforcomp:d[`procs] except params`compproc;
   b:d[`results] where d[`procs]=params`compproc;                                                                /- obtain the check to compare the others to
-  
-  if[all 0W=first b;                                                                                            /- if error in compare proc then fail check
+
+  if[@[{all 0W=x};first b;0b];                                                                                  /- if error in compare proc then fail check
     .dqe.updresultstab[runtype;idnum;.z.p;0b;"error: error on comparison process";`failed;params;`];:()];
-  errorprocs:d[`procs] where all each 0W=d`results;
+  errorprocs:d[`procs] where (),all each @[{0W=x};d`results;0b];
```

Comment:
Listing needed?

Reply:
The listing needs to be there because of the where getting a type error when the error trap returns 0b. 
Was there originally for if only one proc returns an error but think that might of been fixed elsewhere.